### DISCUSSION TITLE
Save as author folder

### DISCUSSION THREAD

**Discussion by ghost** [BlafKing/sd-civitai-browser-plus#120] [Ideas]
It is possible to have an option to use the author name as sub folder? Thank
Personally I use a format like that "...\Lora\Char\[author name]\model name.safetensors" if it's possible to add the option to create or add in the existing [author name] folder that pure qol :)

**Comment by BlafKing** [comment]
Sounds good! I'll add that to the automatically inserted sub folders list! :)

**Comment by BlafKing** [comment]
This has now been added in v3.1.0! :)
